src/components/ModalPayment/ModalCardDetails/ModalCardDetails.tsx

```diff
@@ -421,6 +426,12 @@ const ModalCardDetails = ({
             cardNonceResponseReceived={cardNonceResponseReceived}
             formId="SPF"
             apiWrapper=""
+            inputEventReceived={() => {
+              console.log('??', checkFormValidity());
+              if (checkFormValidity()) {
+                setCanSubmit(true);
```

Comment:
any reason not to do `setCanSubmit(checkFormValidity())` ?

Reply:
no; not particularly. can be condensed. good catch